### ISSUE TITLE
pane synchronize

### ISSUE THREAD

**Issue by sandikata** [wez/wezterm#2658]
Hello,

Is it supported in wezterm a synchronized panes (like it tmux) ?

Example of tmux.conf for synchronize panes.

```
# toggle synchronize-panes
bind C-x setw synchronize-panes
setw -g window-status-current-format '#{?pane_synchronized,#[bg=red],}#I:#W'
setw -g window-status-format         '#{?pane_synchronized,#[bg=red],}#I:#W'
```

Regards.

**Comment by wez** [comment]
No, wezterm doesn't have this feature

**Comment by liuyongsheng** [comment]
hope to support

**Comment by kaddkaka** [comment]
In a similar vein, scrolling multiple panes in sync like [vim's scrollbind option](https://vimhelp.org/scroll.txt.html#scroll-binding) would be nice to have sometimes.

**Comment by alexandrestein** [comment]
Any plan to support those features at some point ?
I would appreciate them. :smiley:

**Comment by wez** [comment]
I personally have no plans to implement this any time soon.

**Comment by Felixoid** [comment]
I wanted to launch multiple commands simultaneously on different servers over SSH, and it's surprisingly impossible.

I understand it's a personal preference, but there are plenty of terminals and multiplexers doing it:

- [terminator](https://github.com/gnome-terminator/terminator/issues/623)
- [kitty](https://sw.kovidgoyal.net/kitty/kittens/broadcast/)
- `synchronize-panes` in tmux
- pretty poor implementation, but even [screen](https://stackoverflow.com/a/6514516/3706827) does it

Basically, it's quite a daily tool for sysadmins to run commands simultaneously over many servers. Sure, there are other tools for it, although, it's a good thing to have in a terminal emulator.

**Comment by Felixoid** [comment]
Dear @wez, operating with a zoo of panes like on the screenshot is a very complex task without parallel input into the tab v_v

![image](https://github.com/wez/wezterm/assets/3025537/f4ed0e0c-44ac-4b8f-b2e6-fbbc32971308)

**Comment by peterjaric** [comment]
My current workaround is to run [Byobu](https://www.byobu.org/) in a Wezterm tab. Not perfect, but gets the work done. Ctrl-F2 and Shift-F2 to create splits and Alt-F9 to toggle sending input to all splits.

**Comment by daUnknownCoder** [comment]
> zoo of panes like on the screenshot is a very complex task without parallel input into the tab

use Warp terminal?

**Comment by michbsd** [comment]
+1 from here - this would indeed be a welcome feature ... :-)

**Comment by ricardoseriani** [comment]
This will really be a welcome feature.
It will make more pleasant and much easier when using multiple panes.
I hope that this get implemented.
BTW, thanks for this amazing project 🤩.

**Comment by anandubey** [comment]
@wez  I would like to help implementing this feature. Any guide on how to get started?

**Comment by annie444** [comment]
+1

**Comment by anshu3012** [comment]
Its 2026 and still crickets..

Would appreciate this feature but if you don't have the bandwidth then I would love to contribute, @wez

**Comment by jceloria** [comment]
How is this helpful? You can always contribute if you don't like the speed
of which issues are addressed.

**Comment by annie444** [comment]
I finally got around to making a plugin for this: [annie444/sync-panes.wez](https://github.com/annie444/sync-panes.wez)

Contributions are welcome!

**Comment by evs-xsarus** [comment]
Thanks for the work. Plugin does as advertised!
